test/config/logs.go

```diff
@@ -22,6 +22,9 @@ var (
 	// TestLogFileName is the file name to dump `TestLogWriter` content when
 	// test finish
 	TestLogFileName = "test-output.log"
+
+	// JenkinsLogFilename is the file name where the Jenkins output is saved
+	JenkinsLogFilename = "Jenkin-output.log"
```

Comment:
Jenkin --> jenkins

Reply:
Can you be more descriptive than 'Jenkins output'? What exactly constitutes 'Jenkins output'?

Reply:
Hi, it's a leftover. I'll delete this one, sorry!